.buildkite/pipeline.yml

```diff
@@ -26,3 +26,18 @@ steps:
     command: |
       cp gradle.properties-example gradle.properties
       ./gradlew lintWordpressVanillaRelease
+  - label: "Test WordPress"
+    <<: *docker-container
+    command: |
+      cp gradle.properties-example gradle.properties
```

Comment:
The reason we repeat this `cp` command on every step is because they all run in parallel and in a dedicated container and there's no explicit checkout because Buildkite does it automatically.

I haven't reviewed many Buildkite PRs so far and it's been a while since I worked with it. Can you confirm that my understanding is correct, please? 😄

Reply:
Yes, checkout is handled by Buildkite and every job currently runs in parallel, meaning separate VMs and Docker containers, so they all need to copy gradle properties. I will propose dropping gradle.properties completely in an internal RFC soon, so hopefully we'll get rid of these steps.

> I haven't reviewed many Buildkite PRs so far and it's been a while since I worked with it. 

This is the second Buildkite PR, so you haven't missed much :)